### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by FxKu**
Will look like this:

## Checklist

Thanks for submitting a pull request to the Postgres Operator project.
Please, ensure your contribution matches the following items:

- [ ] Your go code is [formatted](https://blog.golang.org/gofmt). Your IDE should do it automatically for you.
- [ ] You have updated [generated code](https://github.com/zalando/postgres-operator/blob/master/docs/developer.md#code-generation) when introducing new fields to the `acid.zalan.do` api package.
- [ ] New [configuration options](https://github.com/zalando/postgres-operator/blob/master/docs/developer.md#introduce-additional-configuration-parameters) are reflected in CRD validation, helm charts and sample manifests.
- [ ] New functionality is covered by [unit](https://github.com/zalando/postgres-operator/blob/master/docs/developer.md#unit-tests) and/or [e2e](https://github.com/zalando/postgres-operator/blob/master/docs/developer.md#end-to-end-tests) tests.
- [ ] You have checked existing open PRs for possible overlay and referenced them.

### CODE REVIEW

**Comment by sdudoladov** [comment]
:+1:

File: .github/PULL_REQUEST_TEMPLATE/postgres-operator-pull-request-template.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Thanks for submitting a pull request to the Postgres Operator project.
```
Comment (Jan-M) [inline .github/PULL_REQUEST_TEMPLATE/postgres-operator-pull-request-template.md:1]: We could also have something like

```
# Problem description

# Checklist
```

to make it look and feel a bit nicer

**Comment by sdudoladov** [comment]
:+1:

**Comment by FxKu** [comment]
:+1:

**Pull request merged**